data/hai/hai outfits.txt

```diff
@@ -105,6 +105,71 @@ effect "ion impact"
 	"velocity scale" -.05
 
 
+outfit "True Unfettered License"
+	category "Special"
+	"capture attack" 10
+	"capture defense" 10
+	"quantum keystone" 1
+	thumbnail "outfit/true unfettered license"
+	description "This sword may be bit of a strange license, but whenever I look at it I feel pride, and whenever the Unfettered look at it I see respect in their eyes. This was given to me by one of their warlords as a sign of trust. It is quite handy in fights too, thanks to the special properties of the keystone it is crafted from. It has some useful features, and legends say it allows the user to phase through matter for a short period of time."
```

Comment:
This description isn't final, but I changed it to not be first person, I guess I did it that way because of how personal this item should be to the player. (also I'm not that familiar with how it usually goes in ES, thanks for pointing this out)